### PR TITLE
[DOCS-6564] Add monitoring dashboard

### PR DESCRIPTION
**Pull request by oliveralfresco2018**

### CODE REVIEW

File: process-automation/latest/admin/release.md [hunk 1]
```diff
@@ -147,6 +147,20 @@ The **Undeploy** action removes a deployed application.
 
 The **Manage Permissions** option allows for additional users to be granted user or administrator access to an application without having to upgrade it.
 
+### Monitoring
+
+You can monitor the health of your installation by using the Monitoring dashboard. There is a dashboard for each deployed application.
+
+To access the dashboard.
```
Comment (andrea-ligios) [inline process-automation/latest/admin/release.md:154]: I guess this should end with a colon

File: process-automation/latest/admin/release.md [hunk 2]
```diff
@@ -147,6 +147,20 @@ The **Undeploy** action removes a deployed application.
...
+
+1. Log into the Alfresco Admin App.
+
+2. Expand **Devops** from the left pane and select **Application Instances**.
+
+3. Click the three dots next to the application you want to monitor and select **Monitoring**.
+
+You will see the Monitoring dashboard for the application. Services that are operational have a green dot. Services that are operational but may have some errors have a yellow dot. Services that are not operating and require assistance have a red dot.
```
Comment (andrea-ligios) [inline process-automation/latest/admin/release.md:162]: I would change `some errors` with `some problems`. 
For example, a service that is loading has no errors but it has temporary problems in being accessed until the loading has been completed.

File: process-automation/latest/admin/release.md (hunk 1 above)
Comment (andrea-ligios) [reply process-automation/latest/admin/release.md:154]: I see this is done in a mixed way with a colon for bullet lists and a dot for ordered lists... never mind then.

**Review by andrea-ligios — APPROVED** [review]
LGTM!

**Review by gravitonian — APPROVED** [review]
Built and looked at the page. Looks good.

**Pull request merged**